frontend/src/components/University/MainUniversitySidebar.jsx

```
@@ -18,30 +19,33 @@ export default function UniversitySidebar({ onDeptClick }) {
+        className="flex items-center gap-2 text-blue-700 text-lg font-semibold cursor-pointer hover:underline"
+        onClick={() => navigate(`${uni.name}`)}
+      >
+        <Building2 size={18} /> {uni.name}
+      </h2>
+      <ul className="ml-1 mt-3 space-y-2">
+        {uni.departments.map((dept) => (
+          <li
+            key={dept}
+            className="flex items-center gap-2 text-sm text-gray-700 cursor-pointer hover:text-blue-500"
+            onClick={() => onDeptClick(uni.name, dept)}
           >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)